src/GrpcProtos/capture.proto

```diff
@@ -139,6 +139,148 @@ message ApiEvent {
   fixed64 r5 = 9;
 }
 
+message ApiScopeStart {
+  // NextID: 16
+
+  int32 pid = 1;
+  int32 tid = 2;
+  uint64 timestamp_ns = 3;
+
+  // We encode the first 64 characters of the scope's name/label using fixed64
+  // values, in order to avoid an additional allocation (for the internal
+  // vector) for short strings. Those allocations has shown a huge performance
+  // impact.
+  // If the name exceeds the 64 characters, the `additional_encode_name` will
+  // be used to encode the additional characters.
+  fixed64 encoded_name_01 = 4;
+  fixed64 encoded_name_02 = 5;
+  fixed64 encoded_name_03 = 6;
+  fixed64 encoded_name_04 = 7;
+  fixed64 encoded_name_05 = 8;
+  fixed64 encoded_name_06 = 9;
+  fixed64 encoded_name_07 = 10;
+  fixed64 encoded_name_08 = 11;
+  bytes additional_encoded_name = 12;
```

Comment:
I would make `additional` a suffix to have a consistent prefix with `encoded_name`. Here and in the other messages.

Reply:
Done.